x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.tsx

```diff
@@ -200,16 +214,28 @@ const INACTIVE_LABEL = i18n.translate(
   { defaultMessage: 'OK' }
 );
 
+function getActionGroupName(alertType: AlertType, actionGroupId?: string): string {
+  actionGroupId = actionGroupId || alertType.defaultActionGroupId;
+  const actionGroup = alertType?.actionGroups?.find(
+    (group: ActionGroup) => group.id === actionGroupId
+  );
+  return actionGroup ? actionGroup.name : 'Unknown';
```

Comment:
You're right, I think it's unlikely. The only way I could see it happening is if somehow the default action group id didn't exist in the action group list...which also shouldn't happen.

Reply:
I played around with it a little bit and if the selected action group id doesn't exist in the action group list, the alert will fail to be created. If I create the alert and then change the action group ids in the action group list, then it might lead to the action group being not found. I could see a scenario where the action groups might change for an alert type, but the older event logs contain an old action group id, so I think it's better to err on the safe side but I agree, it's better to just not show it than show 'Unknown'.

Reply:
I think not showing the label vs showing "Unknown" is right.

I haven't looked at the code yet, so context may not be right, but could this happen if we find an event from a previous version without the action group in the event?  I'm worried, based on the recent issue we had with newly added fields being undefined after a half-done migration (the dev o11y server), with depending on values being set from es docs, when there's a chance they won't be.

Reply:
Updated to not show the label.